### ISSUE TITLE
DLL causing CTD:  V1.6.1170

### ISSUE THREAD

**Issue by TheBigWillie** [pWn3d1337/Skyrim_SpellHotbar2#9]
Not sure what is going on but Spell Hotbar 2 will not work with the current dll for many users. It causing crashes. Even after a clean save.

**Comment by pWn3d1337** [comment]
Please provide more information. Skyrim game version and post the please logfile. You are the first one to report a crash yet.

**Comment by TheBigWillie** [comment]
Sure. 

The symptoms I experience is when I load the DLL file, the game CTDs. I got one workaround when I managed to install ONLY the core files, activate the esp first, then adding the dll file, but thats it for me. The problem is when I tried to load the optional files for the compatible spells, or anything else, I get a CTD with the attached crashlog below.

Another flag note is I do not use Mod Managers since I really don't rely on them, when they glitch (and they do), they can really do some damage. I manually install all mods.

Here's the log
[crash-2024-10-25-21-04-38.log](https://github.com/user-attachments/files/17530726/crash-2024-10-25-21-04-38.log)

Hope it helps.

**Comment by pWn3d1337** [comment]
Looks like the reading of spell data files caused the crash
`SpellHotbar::SpellDataCSVLoader::load_spell_data(class std::filesystem::path)` which happens AFTER the icon loading. Are you sure you did not mess up your install ?  Also please show me your Spellhotbar2.log located in 'Documents\My Games\Skyrim Special Edition\SKSE'

**Comment by TheBigWillie** [comment]
Yeah, I see it in the SKSE log looks like specifically the dual casting perk data. Attached below.

[SpellHotbar2.log](https://github.com/user-attachments/files/17535623/SpellHotbar2.log)

**Comment by TheBigWillie** [comment]
I found and corrected the issue. 

I was looking at the Scripts folder and realized there had to be a Source for the pex files, so I went ahead and snagged the src files too and yes, I borked the install by not using the font and Conditional files. 

Thanks for the assist.

**Comment by TheBigWillie** [comment]
Yes, it is mainly looking for a "dual cast perk data" but there is none.
Unless it is from that Conditional 9000 directory. I have no idea where to
place them.

**Comment by TheBigWillie** [comment]
Nevermind, it would be under a perkdata directory which I had to create. Now I got it to work.

**Comment by pWn3d1337** [comment]
So it was an install issue, but I will add better exception handling for the next version

**Comment by TheBigWillie** [comment]
No it was my error. I had to figure out which content goes in which
directory. I know now. For directory 9000 though, manual installation
requires a perkdata folder to be created.
